compiler/parser/src/token.rs

```diff
@@ -240,6 +240,23 @@ pub enum StringKind {
     Unicode,
 }
 
+impl TryFrom<String> for StringKind {
+    type Error = String;
+
+    fn try_from(value: String) -> Result<Self, String> {
+        match value.as_str() {
+            "" => Ok(StringKind::String),
```

Comment:
does this ever get hit? `lex_identifier` is called when the starting char is some character that starts identifiers so the cases for `r`, `u`, `b`, `f` etc to pass through.

Reply:
You're right. It never hits. I'll remove it.